lib/generators/blacklight/templates/config/initializers/blacklight_initializer.rb

```diff
@@ -2,8 +2,4 @@
 # functionality, for example in Refworks export of Bookmarks. In Rails 4, Blacklight
```

Comment:
Can we clean this up even more? We should use Rails' SECRET_KEY_BASE to seed `Blacklight.secret_key`, and not bother exposing any special configuration.